Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -218,6 +219,37 @@ public boolean onTouch(View v, MotionEvent event)
                     ButtonState.PRESSED);
           }
           break;
+        case MotionEvent.ACTION_MOVE:
+          // For Drum Overlay Button pressed immediately after touch pointer enter bound
```

Comment:
Why? This doesn't match how playing on an actual drum controller works.

Reply:
it because we playin on the overlay, when there are 2 different buttons consecutively, we just slide the finger

Reply:
That sounds much harder to time correctly than tapping, and the drum charts in the games are not designed in a way that would make this action useful most of the time (there is a lot more of hitting the same pad than moving your hand to an adjacent pad). I feel like the usefulness doesn't outweigh the risk of people accidentally hitting two pads when they meant to hit one.

Reply:
yes it rarely used, and mostly used when we hold phone with both hand and only using our two thumb. 
i dont mind if its removed